conductor/lib/common.py

```diff
@@ -574,14 +573,32 @@ def load_resources_file():
 
 def get_conductor_instance_types(as_dict=False):
     '''
-    Get the list of available instances types from the resources.yml file
+    Get the list of available instances types.
     '''
-    resources = load_resources_file()
-    instance_types = resources.get("instance_types") or []
+    from conductor.lib import api_client
+    api = api_client.ApiClient()
 
-    if as_dict:
-        return dict([(instance["name"], instance) for instance in instance_types])
+    from conductor.lib.downloader import get_bearer_token
+    bearer = get_bearer_token()
 
+    account_id = api_client.account_id_from_jwt(bearer.value)
+
+    payload, response_code = api.make_request('api/v1/instance-types',
+                                              use_api_key=True)
+    data = json.loads(payload)
+    if not (response_code == 200 or data):
```

Comment:
do you mind following the error-checking pattern that the other api helper-functions follow, e.g. 
```python
    payload, response_code = api.make_request('api/v1/instance-types',
                                               use_api_key=True,
                                               raise_on_error=False)   # Note this flag

    if response_code not in (200, ):
        msg = "Failed to get instance types"
        msg += "\nError %s ...\n%s" % (response_code, response)
        raise Exception(msg)

    instance_types = json.loads(response).get("data", [])
```

Using this construct allows us to print out the body of the failed request (e.g. when you get a 422 ...and the body has information that tells you specifically what's wrong with the request).  

Also note that we don't attempt to parse the json until we can verify a success code.  Otherwise, you may try to parse a body that isn't actually json (therefore creating an unhelpful traceback)